src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -44,6 +49,18 @@ public class HttpUtil {
 
   private static CloseableHttpClient httpClient;
 
+  private static final int DEFAULT_CONNECTION_TIMEOUT_MINUTES = 1;
+  private static final int DEFAULT_HTTP_CLIENT_SOCKET_TIMEOUT_MINUTES = 5;
+
+  private static final int DEFAULT_MAX_CONNECTIONS_PER_ROUTE = 100;
+  private static final int DEFAULT_MAX_CONNECTIONS = 100;
+
+  private static final int DEFAULT_TTL_CONNECTION_MINUTES = 2;
+
+  private static final long MONITOR_THREAD_INTERVAL_MS = TimeUnit.SECONDS.toMillis(5);
```

Comment:
can you add a comment about what this is monitoring? also consider making the name more specific.